screen-admin.php

```diff
@@ -173,6 +173,20 @@ function ass_admin_notice() {
 If you feel this service is being misused please speak to the website administrator.', 'bp-ass' );
 
 			$user_ids = BP_Groups_Member::get_group_member_ids( $group_id );
+			$admin_info = bp_core_get_core_userdata( bp_loggedin_user_id() );
+
+			$email_tokens = array(
+				'ges.subject'  => stripslashes( strip_tags( $subject ) ),
+				'usermessage'  => $message,
```

Comment:
Change this to `$notice` since the wrapper text is already added to the default `bp-ges-notice` email content.

Then we can remove the entire `$message` variable as well (lines 160-173).

Reply:
Should we continue to send `$message` through for plain-text fallback?

Reply:
Ahh, yes, we do need to keep `$message` for those that might not be using `bp_send_email()`.

After we make the change to `{{usermessage}}`, that should be all that is necessary.